README.md

```diff
@@ -71,7 +71,7 @@ the `ScheduledSparkApplication` CRD and the corresponding CRD controller.
 
 Spark Operator currently supports the following list of features:
 
-* Supports Spark 2.3 and up.
+* This branch specifically supports Spark 2.2 fork with Kubernetes Support (https://github.com/apache-spark-on-k8s/spark) to add support for PySpark. This branch will be deprecated once PySpark support is added in Spark 2.4.
```

Comment:
nit: change the format to `[the Spark 2.2 fork with Kubernetes support](https://github.com/apache-spark-on-k8s/spark)`.